fury/ui.py

```diff
@@ -3124,8 +3124,8 @@ class Checkbox(UI):
     ----------
     labels : list(string)
         List of labels of each option.
-    options : list(Option)
-        List of all the options in the checkbox set.
+    options : dict{label:Option}
```

Comment:
here, we want to keep a list or list of tuple. Then we convert these input to a dictionary.